tests/Functional/src/com/microsoft/aad/adal/test/AuthenticationContextTest.java

```diff
@@ -1906,6 +1906,70 @@ private AuthenticationContext getAuthenticationContext(Context mockContext, Stri
 
         return context;
     }
+    
```

Comment:
May be picky, but can we have a component test? pass in the client id, redirect uri and all the other required parameters, and then call acquiretoken, make sure if it went through the broker flow and we get the exception with the correct message back when the redirect uri is not right.